samples/net/wifi/shell/socs/esp32_procpu.conf

```diff
@@ -1,11 +1,2 @@
-CONFIG_WIFI=y
-
-CONFIG_NETWORKING=y
 CONFIG_NET_L2_ETHERNET=y
```

Comment:
Wi-Fi won´t  work without `CONFIG_NET_L2_ETHERNET=y`. I guess we can `select NET_L2_ETHERNET`in the driver kconfig it self.

Reply:
I was hoping you would say this :) yep, sounds like a good idea!